backend/controller/dal/lease.go

```diff
@@ -96,7 +96,7 @@ func (d *DAL) AcquireLease(ctx context.Context, key leases.Key, ttl time.Duratio
 			return nil, nil, fmt.Errorf("failed to marshal lease metadata: %w", err)
 		}
 	}
-	idempotencyKey, err := d.db.NewLease(ctx, key, sqltypes.Duration(ttl), pqtype.NullRawMessage{RawMessage: metadataBytes})
```

Comment:
I think this was my fault! Might be an idea to check for other locations where I did this  😬

Reply:
Made another issue for that 👍

Reply:
I just checked, looks like this was the only location.